glean-core/rlb/src/lib.rs

```diff
@@ -151,6 +151,17 @@ where
     f(&mut lock)
 }
 
+/// Launches a new task on the global dispatch queue with a reference to the Glean singleton.
+pub fn launch_with_glean(callback: impl FnOnce(&Glean) + Send + 'static) {
+    dispatcher::launch(|| crate::with_glean(callback));
+}
+
+/// Launches a new task on the global dispatch queue with a mutable reference to the
+/// Glean singleton.
+pub fn launch_with_glean_mut(callback: impl FnOnce(&mut Glean) + Send + 'static) {
```

Comment:
```suggestion
fn launch_with_glean_mut(callback: impl FnOnce(&mut Glean) + Send + 'static) {
```

Reply:
Yep that makes sense! Thank you for reviewing.